feature: Breadcrumbs inspired by Dinside.no

egreb opened the pull request:
Content and color is possible props.

#### Please tick a box ###
- [x] **feature** _(A new feature_)
- [ ] **fix** _(A bug fix_)
- [ ] **refactor** _(A code change that neither fixes a bug nor adds a feature_)
- [ ] **docs** _(Documentation only changes_)
- [ ] **performance** _(A code change that improves performance_)
- [ ] **style** _(Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)_)
- [ ] **build** _(Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)_)
- [ ] **ci** _(Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)_)
- [ ] **test** _(Adding missing tests or correcting existing tests_)

#### If you're adding a feature, is it documented in a storybook story?
- [x] Yes
- [ ] No
- [ ] Not adding a new feature

#### Are the components you're working on reusable between brands?
- [x] Yes _(Using variables that are defined in the default theme)_
- [ ] No _(I hard coded some values)_
- [ ] Not working on any components

#### If you're creating markup, did you add proper semantics? 
- [ ] I added [ARIA attributes](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA)
- [ ] I added microdata from [Schema.org](http://schema.org/)
- [ ] Semantics are derived from HTML
- [ ] Not applicable

_(Did you do a CR and see that there is something that we should check for each PR, that are not on the list, please [update this document](https://github.com/dbmedialab/shiny/edit/master/.github/PULL_REQUEST_TEMPLATE.md))_

egreb committed:
Breadcrumbs inspired by Dinside.no
Content and color is possible props.

misund committed:
Merge branch 'master' into feature/breadcrumbs

misund commented on src/atoms/Breadcrumbs/BreadcrumbItemBase.js:
```diff
@@ -0,0 +1,148 @@
+import styled, { css } from 'react-emotion';
```
We can delete this entire file.

misund commented on src/molecules/BreadcrumbBar.js:
```diff
@@ -0,0 +1,91 @@
+import React from 'react';
```
This file is not used in the story. We _can_ delete it.

If we want to keep it - and I think that maybe we should? – maybe something like ...
```jsx
import { HorizontalLinkBar } from './HorizontalLinkBar'
import { BreadcrumbItem } from './BreadcrumbItem'

const BreadcrumbBar = props => <HorizontalLinkBar ListItemElement={BreadcrumbItem} {...props} />

export { BreadcrumbBar }
```
... will be enough?

And use it in the story as well.

egreb committed:
Refactored
Refactored due to the comments from @misund, but I changed it away from
using the different `Link` components because breadcrumbs is going to look
and behave different enough from regualar links to justify that breadcrumbs should have its
own components. (long sentence, sorry)
